pubspec.yaml

```diff
@@ -26,7 +26,6 @@ dependencies:
       url: https://github.com/moovida/smashlibs.git
       ref: 399f9c3
 
-  
```

Comment:
@nobnisai @mbasa  
This line seems not to be necessary, so I will delete this line.

Reply:
Okay, got it.

Reply:
Not delete, but added in the new commit.